TWLight/users/templates/users/user_collection_tile.html

```diff
@@ -93,7 +93,7 @@
       {% else %}
         {% if user_collection.auth_latest_sent_app %}
           {% if not user_collection.auth_has_expired %}
-            <a href="{{ user_collection.partner_access_url }}" class="btn btn-sm twl-btn access-apply-button"
+            <a href="{{ user_collection.partner_access_url }}" class="btn btn-sm twl-btn access-apply-button col"
```

Comment:
Instead of adding a `col` class here, add a div that wraps this element with a `col-12` class. Please check out https://getbootstrap.com/docs/4.0/layout/grid/ for more information